pom.xml

```diff
@@ -243,10 +243,9 @@
             <version>1.0.24</version>
         </dependency>
         <dependency>
-            <groupId>org.kohsuke</groupId>
-            <artifactId>trilead-putty-extension</artifactId>
-            <version>1.2</version>
-            <scope>runtime</scope>
+            <groupId>com.google.code.findbugs</groupId>
```

Comment:
Was this a dependency lost as part of the upgrades?

Reply:
This is required to update the parent pom to 4.42; the Job class now requires nullability annotation be present so I had to include this dependency to facilitate that

Reply:
In the previous version of the parent pom this was a direct dependency, actually it was the spotbugs version. Is spotbugs no longer a dependency of the parent? If it is not, can we use the spotbugs one instead of the findbugs one?

Reply:
We should not use the `findbugs` annotations here. It has moved from a direct dependency in the parent pom to the `bom` so, we should use the Jenkins supplied one like so:
```suggestion
            <dependency>
            <groupId>com.github.spotbugs</groupId>
            <artifactId>spotbugs-annotations</artifactId>
            <!--<version>managed by bom</version>-->
        </dependency>